client/cli/go/cmds/device.go

```diff
@@ -322,30 +324,83 @@ var deviceDisableCommand = &cobra.Command{
 }
 
 var deviceResyncCommand = &cobra.Command{
-	Use:     "resync [device_id]",
+	Use:     "resync [object_id]",
 	Short:   "Resync storage information about the device with operation system",
 	Long:    "Resync storage information about the device with operation system",
-	Example: "  $ heketi-cli device resync 886a86a868711bef83001",
+	Example: `  $ heketi-cli device resync 886a86a868711bef83001 `,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		//ensure proper number of args
 		s := cmd.Flags().Args()
+
+		//ensure proper number of args
 		if len(s) < 1 {
-			return errors.New("device id missing")
+			return errors.New("object/cluster id missing")
 		}
 
-		// Set node id
-		deviceId := cmd.Flags().Arg(0)
+		// set objectId
+		objectId := cmd.Flags().Arg(0)
+		// determine operation mode: CLUSTER od DEVICE
+		resyncCluster, err := cmd.Flags().GetBool("cluster")
+		if err != nil {
+			return err
+		}
 
 		// Create a client
 		heketi, err := newHeketiClient()
 		if err != nil {
 			return err
 		}
 
-		//set url
-		err = heketi.DeviceResync(deviceId)
-		if err == nil {
-			fmt.Fprintf(stdout, "Device %v updated\n", deviceId)
+		// a single device Id was specified and no cluster resync was requested
+		if resyncCluster == false {
+			fmt.Println("No cluster-wide resync operation requested. Using object_id as device_id.")
+			//set url
```

Comment:
This looks like a stray comment.

Reply:
Yes, it's definitely a stray comment, I'll remove that

Reply:
Unfortunately, it still looks like it exists.

Reply:
I can't see it anymore, I checked in my fork too, it's not there anymore

Reply:
Strange. I see it when I view "all files" on the PR; I also see it when I pull the PR locally and look at the diff:
```
@@ -342,10 +384,18 @@ var deviceResyncCommand = &cobra.Command{
                        return err
                }
 
-               //set url
-               err = heketi.DeviceResync(deviceId)
-               if err == nil {
-                       fmt.Fprintf(stdout, "Device %v updated\n", deviceId)
+               if resyncCluster == false {
+                       //set url                                    <-------- here
+                       err = heketi.DeviceResync(objectId)
+                       if err == nil {
+                               fmt.Fprintf(stdout, "Device %v updated\n", objectId)
+                       }
```

Maybe you need to (force) push?

Reply:
yeah maybe. I just squashed all commits into one and force-push it again.
let's see :)

Reply:
I still see it even in the squashed pr! It's right under the `if resyncCluster == false {` line

Reply:
I re-submitted the PR. Do you see that still?